assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -288,6 +288,8 @@ storage:
           %{~ if controller_count == "1" ~}
           -A INPUT -p tcp --dport 7443 -j ACCEPT
           %{~ endif }
+          -A INPUT -p tcp --dport 9100 -j ACCEPT
```

Comment:
Why do we want to modify these rules, instead of adding a calico global network policy?

I mean, I don't think we need this for bootstrapping or to connect if the cluster is "broken", right?

Reply:
This is because currently it doesn't work. This is why I created #137.

Reply:
But it *does* work for other ports (like 80 and 443), and in during tests I've used it with nodePort services (few months ago)

It seems really weird... unless 80 and 443 open via the calico rules doesn't work? :thinking:

Reply:
I wonder why does it not cause issues on worker nodes and only on controller nodes?

Reply:
Also this won't be a huge problem going forward because node-exporter has been modified to listen on private interface. Now we need to start consuming the latest node-exporter for that.

Reply:
If it worked for workers and not controllers, maybe the selector on the calico rules you tried were matching workers and not controllers? :thinking: 

Do you have the rule handy?

Reply:
See #201 for explanation.

Reply:
Does this mean that we don't need to expose 9100 port anymore to access it internally?

Reply:
If tests pass I don't need to do anything here! I have reverted the two commits which exposed the ports.

Reply:
@surajssd you need to open them in Calico policy instead.

Reply:
I can confirm we don't need any additional changes and things work as they are. Tests have passed, and I can confirm in my manual testing that node-exporter and kubeproxy are reachable now.